src/theory/quantifiers/sygus/cegis_unif.h

```diff
@@ -47,6 +47,10 @@ namespace quantifiers {
 class CegisUnifEnumDecisionStrategy : public DecisionStrategyFmf
 {
  public:
+  // Do not hide the zero-argument version of initialize() inherited from the
```

Comment:
Style question here: should this go beneath the constructor? Also, the comment maybe should be /** */?

Reply:
Are you talking about moving the comment or the `using` line as well? Maybe moving it next to the other version of `initialize()` would be a good idea. The `/** */` comments are actually Doxygen comments. Maybe I can use `/* */` though.